zcash_test_vectors/transaction_v6.py

```diff
@@ -156,8 +159,11 @@ def __init__(self, rand, consensus_branch_id, have_orchard_zsa=True, have_burn=T
             self.issuer = IssuanceKeys(self.isk).ik_encoding
             for _ in range(rand.u8() % 5):
                 self.vIssueActions.append(IssueActionDescription(rand, self.issuer))
-            txid = txid_digest(self)
-            self.issueAuthSig = encode_issue_auth_sig(ZSA_BIP340_SIG_SCHEME, schnorr_sign(txid, self.isk, b'\0' * 32))
+
+            t_inputs = [TransparentInput(nIn, rand) for nIn in range(len(self.vin))]
+            sighash_shielded = signature_digest(self, t_inputs, SIGHASH_ALL, None)
```

Comment:
Var name should be `sighash`. Surly not `_sielded` if we sign only transperant inputs.

Reply:
Done